lib/policy_manager.rb

```diff
@@ -58,6 +58,8 @@ def reasons_not_to_merge(pull_request)
     end
 
     reasons_not_to_merge
+  rescue StandardError
```

Comment:
I'd sooner we raise a custom error type in https://github.com/alphagov/govuk-dependabot-merger/blob/cbb68ccea9b98d6f5aeeb1e3fa264fef39d3db3a/lib/change_set.rb#L24 and then catch that error here. We should avoid catching `StandardError` if possible, as that could swallow other problems.